content/blog/vlan-versus-vxlan.md

```diff
@@ -0,0 +1,93 @@
+This issue would be solved by using a Layer-3 virtual tunnel over the layer-2 network.
+
+A 12-bit VLAN ID is used in the Ethernet data frames to divide the larger Layer 2 network into multiple broadcast domains.  VLANs could be used only for data centers that require fewer than 4094 VLANs. So, scalability issue is resolved using VXLAN.
+
+There could be 24 or 48 servers connected to the Top-of-the-Rack (ToR) switch. Instead of just one MAC address per server link, the switch must learn all the MAC addresses of the virtual machines (VMs) communicating across the servers. This inadequate MAC address table issue would be addressed with the layer3 tunnel over the layer2 frames.
+
+VXLAN is a tunneling mechanism over layer2 ethernet frames. The encapsulation will be known to VTEPs alone and the VMs would not see it.
+
+### VLAN Header in Ethernet Frame
+
+![VLAN Header](/img/picture1.png "VLAN Header")
+
+* The ethernet frame has 4 bytes VLAN tag field to identify the VLAN-ID that the packet originated from. The format of VLAN-tagged frames is defined in IEEE 802.1Q standard.
+* The VLAN TAG has below fields:
+
+  * TPID – By Default VLAN Tagged packets has 0x8100.
+  * Priority – To indicate the 802.1p priority of the frame
+  * CFI – By default it is 0. 0 indicates MAC addresses are in standard format. 1 indicated non-standard format.
+  * VLAN-ID - 12 Bit value to identify the VLAN.
+
+### VXLAN Header in Ethernet Frame
+
+![VXLAN Header](/img/picture2.png "VXLAN Header")
+
+* The entire ethernet frame including DMAC, SMAC and IP-header or IPv6 header and UDP header is encapsulated with four headers:
+  outer ethernet header, outer IP-header, outer UDP Header and VXLAN Header.
+* VXLAN Header has the following fields:
+
+
+  * Flags -  I bit would be set to 1 for a valid VXLAN Network ID (VNI).  The other reserved 7 bits would be set to zero.
+  * VXLAN-ID: This is the 24 bit value used to designate the VXLAN overlay network.
+
+### VLAN and VXLAN Configuration:
+
+To understand VXLAN further, let us go through the configurations of VLAN and VXLAN. 
+Each VNI would be mapped with the VLAN it is carrying.
+
+![VLAN, VXLAN Configuration](/img/picture3.png "VLAN, VXLAN Configuration")
+
+### FAQs
+
+**1. Will VXLAN replace VLAN standard?**
+
+VLAN standard is specified in IEEE 802.1Q standard by IEEE.
+VXLAN standard is specified in RFC 7438 by IETF.
+VXLAN encapsulates the VLAN tagged packet with four outer headers to transmit the packet in IP based network. It does not replace the VLAN standard.
+
+**2. Does VXLAN always have outer VLAN header?**
+
+The outer VLAN header in VXLAN headers is optional. It is based on the deployment scenario.
+
+**3. Does VXLAN only solve the scalability issue?** 
+
+Along with scalability issue, VXLAN is used for having IP based traffic in core network. 
```

Comment:
The second sentence here isn't clear. But maybe you mean this?

"As a VXLAN is used for network segmentation, it solves two major issues; 1) it allows multiple VMs to share a single physical network without seeing each other's traffic for the reuse of IP addresses, and 2) the VNI identifies the scope of the inner MAC frame originated by the individual VM?

I'm not really sure. Could you please revise for clarity?